.github/workflows/build.yml

```
@@ -19,4 +19,126 @@
+        #                 echo "Pipeline failed."
+        #                 exit 1
+        #             fi
+        #         fi
+        #     else
+        #         echo "Failed to check pipeline status. HTTP Status Code: $status_http_code"
+        #         echo "Response: $status_content"
+        #         exit 1
+        #     fi
+        #     # Wait for a while before checking the status again
+        #     sleep 30
+        # done        
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)